style.css

```diff
@@ -528,20 +926,808 @@ li textarea {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    color: #344563;
+    text-decoration: none;
+    font-size: 15px;
+    line-height: 20px;
+    font-weight: 500;
+    transition: all 0.5s ease-in-out;
+  }
+
+  .navigation .desktop-menu a:hover {
+    letter-spacing: 2px;
+    opacity: 0.6;
+  }
+
+  .headline {
+    background-color: #fff;
+    background-image: url("images/Header\ bg@2x.png");
+    background-repeat: no-repeat;
+    background-size: contain;
+    background-position: right;
+    display: flex;
+    flex-direction: column;
+    border-bottom-left-radius: 70px;
+    width: 99vmax;
+    height: 916px;
+    top: 3rem;
+  }
+
+  .headline-text {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 0;
+    gap: 12px;
+    position: absolute;
+    margin-left: 260px;
+    margin-right: 260px;
+    margin-bottom: auto;
+    top: calc(50% - 268px / 2);
+    width: 70%;
+  }
+
+  .primary1 {
+    display: flex;
+    align-items: center;
+    color: #091e42;
+    font-weight: 700;
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    width: 920px;
+    height: 122px;
+    margin: 0;
+  }
+
+  .headline p {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    color: #344563;
+    margin: 0;
+  }
+
+  .social-media {
+    display: flex;
+    flex-direction: column;
+    margin-top: 450px;
+    margin-left: 260px;
+  }
+
+  .social-media p {
+    color: #7f8cff;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    width: 100%;
+    position: absolute;
+    margin-bottom: 142px;
+    padding: 142px;
+    top: 917px;
+  }
+
+  .card-works {
+    margin: 100px auto;
+    border: 1px solid #dfe1e6;
+    border-radius: 24px;
+    background: #fff;
+    width: 98%;
+    height: auto;
+  }
+
+  .card-works-1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+    gap: 22px;
+  }
+
+  .card-works-2 {
+    display: flex;
+    flex-direction: row-reverse;
+    align-items: center;
+    justify-content: flex-end;
+    gap: 22px;
+  }
+
+  .snapshotdesktop1 {
+    width: 40%;
+    height: auto;
+    margin-top: 24px;
+    margin-left: 24px;
+    border-radius: 8px;
+  }
+
+  .snapshotdesktop2 {
+    margin-top: 24px;
+    margin-right: -22px;
+    width: 40%;
+    height: auto;
+    border-radius: 8px;
+  }
+
+  .snapshotdesktop3 {
+    width: 40%;
+    height: auto;
+    margin-top: 24px;
+    margin-left: 24px;
+    border-radius: 8px;
+  }
+
+  .snapshotdesktop4 {
+    margin-top: 24px;
+    margin-right: 22px;
+    width: 40%;
+    height: auto;
+    border-radius: 8px;
+  }
+
+  .card-works-content-1 {
+    margin-top: 24px;
+    margin-right: 0;
+    width: 50%;
+    height: 50%;
+  }
+
+  .card-works-content-2 {
+    margin-top: 24px;
+    margin-left: 0;
+    width: 50%;
+    height: 50%;
+    gap: -48px;
+  }
```

Comment:
- [ ] I think it would be better also to increase the ```padding``` of your cards and make it identical for ```top, bottom, left and right ``` to match that of the Figma design :+1: :+1: 

your design | Figma design
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/200082899-ec7d3b08-7b22-47b6-86e7-40c500100350.png) | ![image](https://user-images.githubusercontent.com/102757126/200082994-9078bd5b-4ee3-4a85-9583-440363964dea.png)